tools/mac_setup.sh

```diff
@@ -14,8 +16,8 @@ brew "cmake"
 brew "git-lfs"
 brew "zlib"
 brew "bzip2"
-brew "rust"
-brew "rustup-init"
+brew "unzip"
+brew "wget"
```

Comment:
what are these for?

Reply:
getting casadi source files. i could probably do away w wget. does osx have curl by default?

Reply:
yeah, or feel free to use git